js/assessments/sentenceLenghtInTextAssessment.js

```diff
@@ -0,0 +1,54 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var countTooLongSentences = require( "./checkForTooLongSentences.js" );
+
+/**
+ * Calculates sentence length score
+ * @param {array} sentences The array containing sentences.
+ * @param {object} i18n The object used for translations.
+ * @returns {object} Object containing score and text.
+ */
+var calculateSentenceLengthResult = function( sentences, i18n ) {
+	var totalSentences = sentences.length;
+	var tooLong = countTooLongSentences( sentences );
+	var maximumPercentage = 25;
+	var percentage = ( tooLong / totalSentences ) * 100;
+	var score = 9 - Math.max( Math.min( ( 6 / 10 ) * ( percentage - 21.7 ), 6 ), 0 );
```

Comment:
Adding a comment to this line that clarifies the calculation will help others if they ever have to edit this file.
Currently it's unclear what the various values represent.